test/object-store/sync/sync_test_utils.cpp

```diff
@@ -442,9 +421,6 @@ struct BaasClientReset : public TestClientReset {
             wait_for_upload(*realm);
             wait_for_download(*realm);
 
-            wait_for_object_to_persist_to_atlas(m_local_config.sync_config->user, app_session, object_schema_name,
-                                                {{pk_col_name, m_pk_driving_reset}, {"value", last_synced_value}});
-
```

Comment:
Was this removed since we're not terminating and re-enabling sync and we don't need to wait for this to persist before terminating?